src/frontend/components/SøknadsSteg/DinLivssituasjon/useDinLivssituasjon.tsx

```diff
@@ -215,15 +216,15 @@ export const useDinLivssituasjon = (): {
         if (skalTriggeEøsForSøker(oppdatertSøker) !== søkerTriggerEøs) {
             settSøkerTriggerEøs(prevState => !prevState);
         }
+        validerAlleSynligeFelter();
```

Comment:
Finnes det noen bedre måte å gjøre dette på? gjøres for å aktivere neste knapp når noe kommer fra en delvis utfylt state.